bagger/src/main/java/gov/loc/repository/bagger/ui/FileChooser.java

```diff
@@ -1,82 +0,0 @@
-package gov.loc.repository.bagger.ui;

```

Comment:
this was just a file I found that wasn't being used at all. Doesn't really have anything to do with #10